vic/drivers/image/Makefile

```diff
@@ -75,6 +75,7 @@ INCLUDES = -I ${DRIVERPATH}/include \
 
 # Uncomment to include debugging information
 CFLAGS  =  ${INCLUDES} ${NC_CFLAGS}  -ggdb -O0 -Wall -Wextra -std=c99 \
+					 -fopenmp \
```

Comment:
Thanks @apcraig - I did a minor refactor and now this is flexible enough to work for RASM without any changes there.